src/msg/consumer/config.go

```diff
@@ -92,5 +93,6 @@ func (c *Configuration) NewOptions(iOpts instrument.Options) Options {
 	if c.ConnectionReadBufferSize != nil {
 		opts = opts.SetConnectionReadBufferSize(*c.ConnectionReadBufferSize)
 	}
+	opts.SetRWOptions(xio.NewOptions())
```

Comment:
Should this be `opts = opts.SetRWOptions(xio.NewOptions())`?

Reply:
Oops nice catch